src/inc/corinfo.h

```diff
@@ -636,6 +636,7 @@ enum CorInfoHelpFunc
     CORINFO_HELP_READYTORUN_CHKCAST,
     CORINFO_HELP_READYTORUN_STATIC_BASE,
     CORINFO_HELP_READYTORUN_VIRTUAL_FUNC_PTR,
```

Comment:
(Also, there is a special checkin process for them - coordinate with the JIT folks.)